src/kvstore/raftex/SnapshotManager.cpp

```diff
@@ -42,7 +42,12 @@ folly::Future<Status> SnapshotManager::sendSnapshot(std::shared_ptr<RaftPart> pa
                                            std::vector<std::string>&& data,
                                            int64_t totalCount,
                                            int64_t totalSize,
-                                           bool finished) mutable {
+                                           SnapshotStatus status) mutable {
+            if (status == SnapshotStatus::FAILED) {
+                LOG(INFO) << part->idStr_ << "Snapshot send failed, the leader changed?";
```

Comment:
if LOG(ERROR) better ?

Reply:
ERROR means users should take care of it.   But here,  we don't need our users to do any special actions. Just dump the information is enough.